tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+import logging
+logging.getLogger("asyncio").setLevel(logging.ERROR)
+# logging.getLogger("lbrynet").addHandler(logging.StreamHandler())
+# logging.getLogger("lbrynet").setLevel(logging.DEBUG)
```

Comment:
what is going on here?

Reply:
Whoops, this should be with the rest of the logger setup